pkgs/applications/kde/kdenlive.nix

```diff
@@ -70,14 +70,25 @@ mkDerivation {
     kpurpose
     kdeclarative
   ];
-  patches = [ ./mlt-path.patch ];
+  # Both MLT and FFMpeg paths must be set or Kdenlive will complain that it
+  # doesn't find them. See:
+  # https://github.com/NixOS/nixpkgs/issues/83885
+  patches = [ ./mlt-path.patch ./ffmpeg-path.patch ];
   inherit mlt;
+  ffmpeg = ffmpeg-full;
   postPatch =
     # Module Qt5::Concurrent must be included in `find_package` before it is used.
     ''
       sed -i CMakeLists.txt -e '/find_package(Qt5 REQUIRED/ s|)| Concurrent)|'
       substituteAllInPlace src/kdenlivesettings.kcfg
     '';
+  # Frei0r path needs to be set too or Kdenlive will complain. See:
+  # https://github.com/NixOS/nixpkgs/issues/83885
+  # https://github.com/NixOS/nixpkgs/issues/29614#issuecomment-488849325
+  postInstall = ''
+    wrapProgram $out/bin/kdenlive --set FREI0R_PATH ${frei0r}/bin/frei0r-1
+    wrapProgram $out/bin/kdenlive_render --set FREI0R_PATH ${frei0r}/bin/frei0r-1
```

Comment:
Do avoid double wrapping I would use `qtWrapperArgs+=(--set FREI0R_PATH ${frei0r}/bin/frei0r-1)` instead.

Reply:
Thanks for the idea! I did it a bit differently now but with the same goal.